lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -76,6 +87,14 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
         build_changelog_sections(changelog_sections)
       end
 
+      def self.platform_changelogs(releases, platform)
+        platform_changelogs = []
+        releases.each do |release|
+          platform_changelogs.push("  * (#{platform} #{release['name']})[#{release['html_url']}]")
```

Comment:
I wonder if we should create a `Release` data holder to contain the data we need instead of passing around the parsed json... I'm ok with this for now though since we follow the same pattern in a lot of places, and we probably don't need that yet.

Reply:
I agree with a `Release` data holder idea at some point! ☝️